requirements/requirements-testing.txt

```diff
@@ -1,2 +1,3 @@
-pytest>6.2.1
-hypothesis>6.30.0
+pytest
+hypothesis
+coverage
```

Comment:
probably need lowest supported versions here OR probably would be good to pin these as well? What do you think?

Also do we need coverage? We don't seem to be using it.

Reply:
We should probably use coverage, I'll add that tomorrow. Pins are set in the pipfiles to ensure there are no user-facing hard pins. This can yield unresolvable environments fairly easily

Reply:
we should set some minimal version in that case?